app.js

```diff
@@ -10,16 +10,32 @@ const upload = multer({
 const dotenv = require('dotenv');
 dotenv.config();
 
-
 app.set('view engine', 'ejs');
 app.use(express.static('static'));
 
+// Mongoose
+const mongo = require('mongodb')
```

Comment:
If we were to use mongoose over MongoDB, we should remove anything that makes a connection through MongoDB. Don't worry about dependencies (in `package.json`), I'll make sure that works. For now just make sure variables and references to MongoDB are removed.

Reply:
I agree, but if we delete MongoDB the develop will break, so we first need to change all the other code that uses the MongoDB client and switch it to Mongoose